frontend/packages/kubevirt-plugin/src/components/create-vm/create-vm.tsx

```diff
@@ -127,7 +132,18 @@ const Footer: React.FC<FooterProps> = ({
             isReview
               ? onFinish
               : isSelectTemplate
-              ? () => withSupportModal(template.variants[0], onNext)
+              ? () => {
+                  workload === 'saphana'
+                    ? withSupportModal(template.variants[0], () =>
+                        history.push(
+                          instantiateTemplateBaseURLBuilderMetadata(
```

Comment:
```suggestion
                          instantiateTemplateBaseURLBuilder(
                              preselectedNamespace,
                              `/${template.metadata.namespace}/${template.metadata.name}`
```

Reply:
maybe put template name and namespace  into consts ?

Reply:
?

Reply:
when trying to enter as consts, it causes the app tp crash when trying to enter the wizard.
I've removed the redundant function.